lib/iris/tests/unit/fileformats/grib/grib1_load_rules/test_grib1_convert.py

```diff
@@ -64,7 +64,8 @@ def assert_bounded_message(self, **kwargs):
                       '_forecastTimeUnit': 'hours',
                       'phenomenon_bounds': lambda u: (80, 120),
                       '_phenomenonDateTime': -1,
-                      'table2Version': 9999}
+                      'table2Version': 9999,
+                      '_originatingCentre': 'xxx'}
```

Comment:
Yes, the handling of a single data point has changed.
The converter makes a scalar coord from the '_originatingCentre' property, which was previously just another Mock.  But now that will fail within '_sanitise_array', so this one must be made explicit.
The existing explicit properties are mostly to get around similar Mock-y problems.
I've explained it a bit more in subsequent commits...